corgi/tasks/managed_services.py

```diff
@@ -0,0 +1,109 @@
+from celery_singleton import Singleton
+from django.db import transaction
+from django.utils import timezone
+
+from config.celery import app
+from corgi.collectors.app_interface import AppInterface
+from corgi.collectors.syft import Syft
+from corgi.core.models import (
+    Component,
+    ComponentNode,
+    ProductComponentRelation,
+    ProductStream,
+    SoftwareBuild,
+)
+from corgi.tasks.common import RETRY_KWARGS, RETRYABLE_ERRORS
+from corgi.tasks.sca import save_component
+
+
+@app.task(base=Singleton, autoretry_for=RETRYABLE_ERRORS, retry_kwargs=RETRY_KWARGS)
+def refresh_service_manifests():
```

Comment:
Let's add the return type hint here so that mypy will check this method instead of ignoring it:
```suggestion
def refresh_service_manifests() -> None:
```